controllers/auth.controller.js

```diff
@@ -33,9 +34,21 @@ const registerUser = asyncHandler(async (req, res, next) => {
       .status(409)
       .json({ message: "User with email or username already exists" });
   }
+
```

Comment:
why this is required ? in this function ?

Reply:
Sir, existed user isme check nahi karenge kya agar naya user register karna chah raha toh ki woh email already present hai database mein?